src/components/Header/TopNavigationBar.tsx

```diff
@@ -110,11 +110,11 @@ function TopNavigationBar() {
                                 </MenuItem>
 
                                 {/* Turn your WSX into liquistake WSX */}
-                                <MenuItem>
+                                {/* <MenuItem>
                                     <Typography variant='body2' color='text.primary'>
                                         <Link to="/stake">Stake WSX</Link>
                                     </Typography>
-                                </MenuItem>
+                                </MenuItem> */}
```

Comment:
[suggestion]

Instead of commenting the `MenuItem` component, delete the code since you noted that it was unnecessary.